src/main.ts

```diff
@@ -59,11 +59,13 @@ async function run(): Promise<void> {
     core.debug(`Full tool exe: ${vswhereToolExe}`)
 
     let foundToolPath = ''
+    let vsIntegrationTollsPath = ''
     const options: ExecOptions = {}
     options.listeners = {
       stdout: (data: Buffer) => {
         const installationPath = data.toString().trim()
         core.debug(`Found installation path: ${installationPath}`)
+        vsIntegrationTollsPath = path.join(vsIntegrationTollsPath, 'VSSDK\\VisualStudioIntegration\\Tools\\Bin\\')
```

Comment:
Did you mean to be:
`vsIntegrationToolsPath = path.join(installationPath,'VSSDK\\VisualStudioIntegration\\Tools\\Bin\\')`

Reply:
Good catch 👍  Should be fixed.